website/src/main/tut/docs/users/configuration.md

```diff
@@ -157,4 +196,10 @@ foo(null)
 
 ```scala
 List(1, "") // scalafix:ok NoInfer.any
-```
\ No newline at end of file
+```
+
+&nbsp;
+
+**Note:** Suppression via comments and `@SuppressWarnings` can be combined in the same source file. Be mindful not to 
+introduce overlaps between the two as it can cause confusion. Scalafix gracefully handles overlaps by given 
```

Comment:
s/gracefully//

Should we add an example here? Might also want to add "it can cause confusion and counter-intuitive behavior".

Reply:
done